cmd/godeb/main.go

```diff
@@ -184,7 +184,7 @@ type tarballSource struct {
 }
 
 var tarballSources = []tarballSource{
-	{"http://golang.org/dl/", "//a/@href[contains(., 'storage.googleapis.com/golang/')]"},
+	{"https://golang.org/dl/", "//a/@href[contains(., 'redirector.gvt1.com/edgedl/go/')]"},
```

Comment:
"//a[@class='download']/@href"

is even better

Reply:
@tgulacsi Thanks, I went with that.